### DISCUSSION TITLE
Where are the examples?

### DISCUSSION THREAD

**Discussion by mallappa** [spacenation/swiftui-sliders#47] [Q&A] [answered]
I see a note in README.md that there are examples in:  /Examples/SlidersExamples.xcodeproj

I have cloned the project, see the source and other files. But, I am unable to see this "Examples" folder. Could someone enlighten me if I am doing something wrong?

Pretty looking sliders. Would love to use them.

**Comment by LucasAbijmil** [comment, marked as answer]
Looking at this [issue](https://github.com/spacenation/swiftui-sliders/issues/25) & [PR](https://github.com/spacenation/swiftui-sliders/pull/43) , it turns out that the example project has been replaced by examples via the preview.
However, if you still want to have access to the example project, I suggest you checkout on this commit: `1fcada2`
